src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -15,32 +16,19 @@ import { AssetsDictionary } from 'utils/dictionaries/assets-dictionary';
 import { LoadableValue } from '../../../components/LoadableValue';
 import { weiToFixed } from 'utils/blockchain/math-helpers';
 import { numberFromWei } from 'utils/blockchain/math-helpers';
-import { ethGenesisAddress } from 'utils/classifiers';
 import { StyledTable } from './StyledTable';
 import { LinkToExplorer } from '../../../components/LinkToExplorer';
-import { eventReader } from 'utils/sovryn/event-reader';
 import { useAccount } from '../../../hooks/useAccount';
-import { useStaking_getStakes } from '../../../hooks/staking/useStaking_getStakes';
-import { useVesting_getVesting } from '../../../hooks/staking/useVesting_getVesting';
-import { useVesting_getTeamVesting } from '../../../hooks/staking/useVesting_getTeamVesting';
-import { useVesting_getOriginVesting } from '../../../hooks/staking/useVesting_getOriginVesting';
 import { TxStatus } from 'store/global/transactions-store/types';
+import { backendUrl } from 'utils/classifiers';
+import { useSelector } from 'react-redux';
+import { selectWalletProvider } from 'app/containers/WalletProvider/selectors';
 
 export function HistoryEventsTable() {
   const { t } = useTranslation();
   const account = useAccount();
-  const getStakes = useStaking_getStakes(account);
-  const vesting = useVesting_getVesting(account);
-  const vestingTeam = useVesting_getTeamVesting(account);
-  const vestingOrigin = useVesting_getOriginVesting(account);
+  const { chainId } = useSelector(selectWalletProvider);
   const [eventsHistory, setEventsHistory] = useState<any>([]);
-  const [eventsHistoryVesting, setEventsHistoryVesting] = useState<any>([]);
-  const [eventsHistoryVestingTeam, setEventsHistoryVestingTeam] = useState<any>(
-    [],
-  );
-  const [eventsHistoryVestingOrigin, setEventsHistoryVestingOrigin] = useState<
-    any
-  >([]);
   const [viewHistory, setViewHistory] = useState(false);
```

Comment:
I am thinking that the naming here is a bit incorrect - we use `viewHistory` as an indicator if the history is loading. Therefore, I'd suggest renaming to `isHistoryLoading`.

Reply:
does make sense